src/components/Alert/__stories__/Alert.stories.tsx

```diff
@@ -12,133 +16,227 @@ export default {
     component: Alert,
 } as Meta;
 
-const title = 'Where will you go, hero?';
-const message = 'Choose wisely: the end of the fairy tale depends on your decision';
-const right = 'To the right (lose the horse)';
-const center = 'Straight (find a wife)';
-const left = 'To the left (CC 235.2)';
+type Story = StoryObj<typeof Alert>;
 
-const stories: AlertProps[] = [
-    {
-        title,
-        message,
-        theme: 'danger',
-        view: 'filled',
-        onClose: () => alert('Close button pressed'),
-        actions: <Alert.Action>{right}</Alert.Action>,
+export const Default: Story = {
+    args: {
+        onClose: action('onClose'),
+        title: 'Where will you go, hero?',
+        message: 'Choose wisely: the end of the fairy tale depends on your decision',
     },
+};
+
+const titleCases: Array<{caseTitle: string; propValue: AlertProps['title']}> = [
     {
-        title: <div dangerouslySetInnerHTML={{__html: '<b>Some html title</b>'}} />,
-        message,
-        corners: 'square',
-        theme: 'danger',
-        view: 'outlined',
-        onClose: () => alert('Close button pressed'),
+        caseTitle: 'Regular title',
+        propValue: Default.args?.title,
     },
     {
-        title,
-        message,
-        view: 'outlined',
-        onClose: () => alert('Close button pressed'),
-        actions: [{text: left}, {text: center}, {text: left}],
+        caseTitle: 'Tag title',
+        propValue: <div dangerouslySetInnerHTML={{__html: '<b>Some html title</b>'}} />,
     },
-    {
-        message,
-        theme: 'info',
-        view: 'filled',
-        actions: (
-            <Alert.Actions>
-                <Alert.Action>{center}</Alert.Action>
-            </Alert.Actions>
-        ),
+];
+
+export const Title: Story = {
+    render: (args) => (
+        <Showcase>
+            {titleCases.map(({caseTitle, propValue}, index) => (
+                <ShowcaseItem key={index} title={caseTitle}>
+                    <Alert {...args} title={propValue} />
+                </ShowcaseItem>
+            ))}
+        </Showcase>
+    ),
+    args: {
+        ...Default.args,
     },
+};
+
+const messageCases: Array<{caseTitle: string; propValue: AlertProps['message']}> = [
     {
-        title,
-        message,
-        theme: 'info',
-        view: 'outlined',
+        caseTitle: 'Regular message',
+        propValue: Default.args?.message,
     },
-    {
-        title,
-        message,
-        theme: 'success',
-        view: 'outlined',
-        actions: (
-            <Alert.Actions>
-                <Button view="action">{right}</Button>
-                <Button view="outlined">{center}</Button>
-                <Button view="flat">{left}</Button>
-            </Alert.Actions>
-        ),
+];
+
+export const Message: Story = {
```

Comment:
This story not differ from default. Let's consider title + message props as default story

Reply:
message story has been deleted
https://github.com/gravity-ui/uikit/pull/1597/commits/90fdc08171b2fac12699892e8f4e3767be5578c6